packages/hash/api/package.json

```diff
@@ -93,7 +72,6 @@
     "@local/eslint-config": "0.0.0-private",
     "@local/tsconfig": "0.0.0-private",
     "@types/app-root-path": "1.2.4",
-    "@types/connect-pg-simple": "4.2.4",
     "@types/cron": "2.0.0",
     "@types/dedent": "0.7.0",
     "@types/express-session": "1.17.5",
```

Comment:
There are a few `@types` packages corresponding to dependencies which have been removed: `express-session`, `json-schema`, `pg`

Reply:
Good point, I’ll look through them